Script/AtomicEditor/hostExtensions/coreExtensions/ProjectBasedExtensionLoader.ts

```diff
@@ -75,7 +75,10 @@ export default class ProjectBasedExtensionLoader extends Atomic.ScriptObject imp
                         if (Atomic.fileSystem.fileExists(path)) {
                             let include = new Atomic.File(path, Atomic.FileMode.FILE_READ);
                             try {
-                                return include.readText();
+                                // add a newline to handle situations where sourcemaps are used.  Duktape
+                                // doesn't like not having a trailing newline and the sourcemap process doesn't
+                                // add one.
+                                return include.readText() + "\n";
```

Comment:
This is something that happens when source maps are used.  The source map index get's appended to the last line of the JS file w/o a newline.  As far as I can tell, this is normal for source maps, however duktape doesn't like it.  This is already being handled in the JSRequire cpp file at:
https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/AtomicJS/Javascript/JSRequire.cpp#L165-L167

so the logic that loads the extensions are doing the same thing.